src/App.js

```diff
@@ -1,5 +1,104 @@
+import { Random, Console } from '@woowacourse/mission-utils';
+
 class App {
-  async play() {}
+  async play() {
+    // 게임 시작 문구 출력
+    Console.print('숫자 야구 게임을 시작합니다.');
+
+    let isFinish = false;
+
+    while (!isFinish) {
+      isFinish = await this.playGame();
+    }
+  }
+
+  async playGame() {
+    const RANDOM_VALUE = this.getRandomNumber();
+    let isSuccess =  false;
+
+    // 3스트라이크 된 경우 isSuccess를 true로 변경하여 게임 종료
+    while (!isSuccess) {
+      try {
+        isSuccess = await this.playOneInput(RANDOM_VALUE);
+      } catch (e) {
+        throw new Error('[ERROR]');
+      }
+    }
+
+    // 게임 종료 후 재시작, 완전 종료 확인
+    const RESTART = await this.startGameAgain();
+    return RESTART;
+  }
+
+  async playOneInput(RANDOM_VALUE) {
+    const RESULT = {
+      strike: 0,
+      ball: 0,
+      nothing: 0
+    };
+    const INPUT_VALUE = await this.getInputNumber();
+    const INPUT_VALUE_ARR = [...INPUT_VALUE];
+
+    INPUT_VALUE_ARR.forEach((num, idx) => {
+      if (num === RANDOM_VALUE[idx]) RESULT.strike++;
+      else if (RANDOM_VALUE.includes(num)) RESULT.ball++;
+      else if (!RANDOM_VALUE.includes(num)) RESULT.nothing++;
+    })
+
+    return this.printResult(RESULT);
+  }
+
+  // 정답으로 사용될 랜덤값 생성
+  getRandomNumber() {
+    const COMPUTER = [];
+    while (COMPUTER.length < 3) {
+      const number = Random.pickNumberInRange(1, 9);
+      if (!COMPUTER.includes(number)) {
+        COMPUTER.push(number);
+      }
+    }
+    return `${COMPUTER[0]}${COMPUTER[1]}${COMPUTER[2]}`;
+  }
+
+  // 세자리 숫자 입력
+  async getInputNumber() {
+    const INPUT = await Console.readLineAsync('숫자를 입력해주세요 : ');
+    this.checkInputNumber(INPUT);
+    return INPUT;
+  }
+
+  // 게임 종료 후 재시작, 완전 종료 확인
+  async startGameAgain() {
+    let input = await Console.readLineAsync('게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.\n');
+    if (input === '1') return false;
+    if (input === '2') return true;
+    else throw new Error('[ERROR] 잘못된 형식입니다.');
+  }
+
+  // 세자리 숫자 예외 처리 - 입력 값이 숫자가 아니거나, 중복을 제거한 길이가 3이 아니면 throw error
+  checkInputNumber(input) {
+    const SET_INPUT = [...new Set(input)];
+
+    if (isNaN(parseInt(input)) || SET_INPUT.length !== 3) {
+      throw new Error('[ERROR] 숫자가 잘못된 형식입니다.');
```

Comment:
혹시 의도하신 게 `중복 제거 -> 길이 구하기`여서 set에서 배열로 자료형을 변경하신 거라면 
```suggestion
    if (isNaN(parseInt(input)) || new Set(input).size !== 3) {
      throw new Error('[ERROR] 숫자가 잘못된 형식입니다.');
```
으로도 가능합니다!!